src/wast-parser.y

```diff
@@ -743,6 +746,23 @@ const_expr :
     instr_list
 ;
 
+/* Exceptions */
+except :
```

Comment:
except vs exception is a bit confusing here, though I'm not sure I have an idea for something better. :-) Thoughts?